CMakeLists.txt

```diff
@@ -551,6 +551,16 @@ add_subdirectory(subsys)
 add_subdirectory(drivers)
 add_subdirectory(tests)
 
+# Add all zephyr modules subdirectories.
```

Comment:
Can we print a message here:
`Including modules: tinycbor, mbedtls, ...`

Reply:
Well, those are not zephyr modules in `west.yml` yet.
Will they become that prior to 1.14 ?

Reply:
I think @carlescufi means "print the list of modules discovered at runtime", not that exact list

Reply:
Ah, got it.
Good point, will do that.

Reply:
Included the print message a couple of lines below.